app/dashboard/trainee/profile.tsx

```diff
@@ -94,30 +81,34 @@ export default function Profile() {
                 profile.avatar
                   ? { uri: profile.avatar }
                   : {
-                      uri: `https://ui-avatars.com/api/?name=${encodeURIComponent(profile.name)}&background=random&color=fff`,
+                      uri: `https://ui-avatars.com/api/?name=${encodeURIComponent(profile?.name || 'Unknown')}&background=cbecd7&size=256`,
                     }
               }
-              className="w-[100] h-[100] rounded-full"
+              className="w-28 h-28 rounded-full"
             />
-            <TouchableOpacity className="absolute right-[-65px] bottom-8 p-1">
-              <SvgXml xml={editPic} />
+            <TouchableOpacity
```

Comment:
Refactor this route

Suggestions: `/dashboard/edit-profile` or `/dashboard/profile/edit`